src/web/crate_details.rs

```diff
@@ -118,6 +119,7 @@ impl CrateDetails {
                 releases.description_long,
                 releases.release_time,
                 releases.build_status,
+                (SELECT id FROM builds WHERE builds.rid = releases.id ORDER BY build_time DESC LIMIT 1) AS latest_build_id,
```

Comment:
I think we only need the latest _successful_ build here.

Reply:
valid point, updated it.